src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AssemblyContigWithFineTunedAlignments.java

```diff
@@ -118,66 +129,103 @@ public final boolean hasOnly2GoodAlignments() {
         return sourceTig.hasOnly2Alignments();
     }
 
-    public final boolean hasIncompletePicture() {
-        if ( hasOnly2GoodAlignments() )
-            return hasIncompletePictureFromTwoAlignments();
-        else
-            return hasIncompletePictureFromMultipleAlignments();
+    public enum AlignmentSignatureBasicTypes {
+        Suspicious, Simple, Complex;
+    }
+
+    public enum Suspicious {
```

Comment:
Rename to `ReasonForAlignmentClassifcationFailure` or something like that, and upcase values.